apiserver/facades/client/modelmanager/modelinfo_test.go

```diff
@@ -178,9 +181,13 @@ func (s *modelInfoSuite) SetUpTest(c *gc.C) {
 
 	s.callContext = context.NewEmptyCloudCallContext()
 
+	ctrl := gomock.NewController(c)
```

Comment:
Same here.

Reply:
done, this mock wasn't needed as well, I created `setUpMocks()` on `modelmanager_test.go` were the mocks are needed.